### ISSUE TITLE
Trouble validating TRAPI 1.3-beta

### ISSUE THREAD

**Issue by edeutsch** [NCATSTranslator/reasoner-validator#32]
Hi, I am trying to validate some TRAPI 1.3-beta messages and they are not validating but I don't understand why.

Here's my test code:
```
import requests
import json
import re
from datetime import datetime
from reasoner_validator import validate
from jsonschema.exceptions import ValidationError

# Set the base URL for the ARAX reasonaer TRAPI 1.1 base
endpoint_url = 'https://arax.ncats.io/api/arax/v1.3'

url = endpoint_url + '/response/54172'
url = endpoint_url + '/response/60320'
print(f"Retrieving response from {url}")

response_content = requests.get(url, headers={'accept': 'application/json'})
status_code = response_content.status_code
if status_code != 200:
    print("ERROR returned with status "+str(status_code))
    print(response_content)
    exit()

# Unpack the response content into a dict
response_dict = response_content.json()
print(f"Retrieved a message with {len(response_dict['message']['results'])} results")
envelope = response_dict

trapi_version = '1.2.0'
trapi_version = '1.3.0-beta'

try:
    validate(envelope,'Response',trapi_version)
    if 'description' not in envelope or envelope['description'] is None:
        envelope['description'] = 'reasoner-validator: PASS'

except ValidationError as error:
    print("Validation failed")
    timestamp = str(datetime.now().isoformat())
    if 'logs' not in envelope or envelope['logs'] is None:
        envelope['logs'] = []
        envelope['logs'].append( { "code": 'InvalidTRAPI', "level": "ERROR", "message": "TRAPI validator reported an error: " + str(error),
            "timestamp": timestamp } )
    if 'description' not in envelope or envelope['description'] is None:
        envelope['description'] = ''
        envelope['description'] = 'ERROR: TRAPI validator reported an error: ' + str(error) + ' --- ' + envelope['description']

print(f"Result: {envelope['description']}")
```

and when I run this, it appears to fail validation. There's a lot of spewage that's a bit difficult to understand, but I think the crux is:
```
Failed validating 'oneOf' in schema['properties']['message']['properties']['results']:
    {'oneOf': [{'description': 'List of all returned Result objects for '
                               'the query posed. The list SHOULD NOT be '
                               "assumed to be ordered. The 'score' "
                               'property,\n'
                               ' if present, MAY be used to infer result '
                               'rankings.',
                'items': {'$ref': '#/components/schemas/Result'},
                'type': 'array'},
               {'type': 'null'}]}

On instance['message']['results']:
    [{'confidence': None,
      'description': 'No description available',
      'edge_bindings': {'N1': [{'attributes': None, 'id': 'N1_2'}],
                        'e00': [{'attributes': None,
                                 'id': 'infores:rtx-kg2:CHEMBL.COMPOUND:CHEMBL112-biolink:physically_interacts_with-UniProtKB:P23219'},
                                {'attributes': None,
                                 'id': 'infores:service-provider-trapi:PUBCHEM.COMPOUND:1983-biolink:physically_interacts_with-NCBIGene:5742'},
                                {'attributes': None,
                                 'id': 'infores:rtx-kg2:UniProtKB:P23219-biolink:physically_interacts_with-CHEMBL.COMPOUND:CHEMBL112'}]},
      'essence': 'PTGS1',
      'essence_category': "['biolink:Protein']",
      'id': None,
      'node_bindings': {'n00': [{'attributes': None,
                                 'id': 'CHEMBL.COMPOUND:CHEMBL112',
                                 'query_id': None}],
                        'n01': [{'attributes': None,
                                 'id': 'UniProtKB:P23219',
                                 'query_id': None}]},
      'reasoner_id': 'ARAX',
      'result_group': None,
      'result_group_similarity_score': None,
      'row_data': [1.0, 'PTGS1', "['biolink:Protein']"],
      'score': 1.0,
      'score_direction': None,
      'score_name': None},
...
```

Any ideas what going on here?

**Comment by edeutsch** [comment]
test code slightly altered to show that validating a TRAPI 1.2 message works fine:
```
import requests
import json
import re
from datetime import datetime
from reasoner_validator import validate
from jsonschema.exceptions import ValidationError

endpoint_url = 'https://arax.ncats.io/api/arax/v1.3'

test = 2

if test == 1:
    url = endpoint_url + '/response/60366'
    trapi_version = '1.2.0'
else:
    url = endpoint_url + '/response/54172'
    url = endpoint_url + '/response/60320'
    trapi_version = '1.3.0-beta'

print(f"Retrieving response from {url}")

response_content = requests.get(url, headers={'accept': 'application/json'})
status_code = response_content.status_code
if status_code != 200:
    print("ERROR returned with status "+str(status_code))
    print(response_content)
    exit()

# Unpack the response content into a dict
response_dict = response_content.json()
print(f"Retrieved a message with {len(response_dict['message']['results'])} results")
envelope = response_dict

try:
    validate(envelope,'Response',trapi_version)
    if 'description' not in envelope or envelope['description'] is None:
        envelope['description'] = 'reasoner-validator: PASS'
    print("Validation succeeded and passed")

except ValidationError as error:
    print("Validation failed")
    timestamp = str(datetime.now().isoformat())
    if 'logs' not in envelope or envelope['logs'] is None:
        envelope['logs'] = []
        envelope['logs'].append( { "code": 'InvalidTRAPI', "level": "ERROR", "message": "TRAPI validator reported an error: " + str(error),
            "timestamp": timestamp } )
    if 'description' not in envelope or envelope['description'] is None:
        envelope['description'] = ''
        envelope['description'] = 'ERROR: TRAPI validator reported an error: ' + str(error) + ' --- ' + envelope['description']

print(f"Result: {envelope['description']}")
```

**Comment by RichardBruskiewich** [comment]
Resolved by release 2.2.13
